### PR TITLE
CME-45 update to java 21

### PR DESCRIPTION
**Pull request by rapolaskaseliscgi**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CME-45

### Change description ###
Update to Java 21

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.0.2",
   "private": true,
   "engines": {
-    "node": "^18.17.0"
+    "node": "^20.11.0"
```
Comment (AntonyLeons) [inline package.json:6]: the reset of the repo is 20.17.0, probably use the latest v20 which is 20.18.0

Comment (rapolaskaseliscgi) [reply package.json:6]: nodejs image on the pipeline has `20.11.x` version installed, and if I request `20.17`, pipeline fails. I'll update to use 20.18 elsewhere.

**Review by AntonyLeons — APPROVED** [review]

**Review by kiran-yenigala-hmcts — APPROVED** [review]
LGTM

**Review by patelila — APPROVED** [review]
Once the version is updated.

**Pull request merged**